unyt/equivalencies.py

```diff
@@ -473,9 +473,9 @@ class EffectiveTemperatureEquivalence(Equivalence):
     effective_temperature: flux <-> temperature
     >>> from unyt import K, W, m
     >>> (5000.*K).to_equivalent('W/m**2', 'effective_temperature')
-    unyt_quantity(35439831.25, 'W/m**2')
+    unyt_quantity(35439828.89119583, 'W/m**2')
```

Comment:
I'm not sure such a large change is warranted (unless it somehow counts as a bug fix), but why is this output changing at all ?